lib/config/providers/pinned_chats_provider.dart

```diff
@@ -0,0 +1,123 @@
+import 'dart:convert';
```

Comment:
It would be nice to add unit tests for new providers

Reply:
Otherwise we will never end with the task having unit tests for all providers... if new untested providers appear 😅